### ISSUE TITLE
deep residual bidir-lstm for har using wearable sensors

### ISSUE THREAD

**Issue by Drug1996** [Drug1996/DeepNAR#8]
### Journals
**Mathematical Problems in Engineering** (2018)
[18_deep_residual_bi_lstm_for_har_using_wearable_sensors.pdf](https://github.com/Drug1996/DeepNAR/files/3317482/18_deep_residual_bi_lstm_for_har_using_wearable_sensors.pdf)
### Author and affiliation
**Author**: _Yu Zhao , Rennong Yang , Guillaume Chevalier , Ximeng Xu , and Zhenxing Zhang_
**Affiliation**: _Aeronautics and Astronautics Engineering College, Air Force Engineering University, Xi'an 710038, China; Laval University, 2325 Rue de l'Universite, Quebec G1V 0A6, Canada_

### Motivation
Aim to enhance the recognition rate for har
### Database
Public database: **UCI, Opportunity**
### Method
- residual network of rnn
![image](https://user-images.githubusercontent.com/32189927/59970090-14064180-9599-11e9-9ace-c51a89f3c245.png)
### Contribution
Proposed a framework called Res-Bidir-LSTM, which improved the accuracy on UCI and Opportunity compared with the previous work they choosed.
<div align=center><img src="https://user-images.githubusercontent.com/32189927/59971581-af0f1380-95b9-11e9-906a-f42e62845b6b.png" width="100%" height="100%"><p>Results of the proposed model for the Opportunity dataset</div>

### Insight
- Borrow the idea from Microsoft's resnet;
- The author mentioned DeepConvLSTM, but didn't compare the proposed model with it, since the proposed model's performance is not better than DeepConvLSTM;
- The model may overfit the test dataset since there was no valid dataset;
### Useful information
- **Use mean and variance normalization for preprocessing;**
- **L2 norm for weight decay and dropout are used to prevent overfitting, and gradient clipping and batch normalization are used to prevent gradient vanishing or explosion as well as overshooting the learning updates;**
- **Grid search for hyper-parameters;**
- **Deal with the missing data;**
